src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -545,6 +589,7 @@ public void track(String eventName, JSONObject properties) {
      * @param eventName the name of the event to send
      */
     public void track(String eventName) {
+        if (hasOptedOutTracking()) return;
```

Comment:
it's fine but you don't need this (track(String eventName, JSONObject properties) will do it)

Reply:
same as above